packages/webamp-modern-2/src/skin/AudioPlayer.ts

```diff
@@ -12,6 +16,14 @@ export class AudioPlayer {
   _eqValues: { [kind: string]: number } = {};
   _eqNodes: { [kind: string]: number } = {};
   _eqEmitter: Emitter = new Emitter();
+  __isStop: boolean = true; //becaue we can't audio.stop() currently
```

Comment:
Why two underscores?

Reply:
I was thinking that single underscore is only relevant for xml attributes?

Reply:
Okay, I just made it single underscore